develop/README.md

```diff
@@ -0,0 +1,32 @@
+# Docker
+
+## Installing ONLYOFFICE Docs
+
+## How to use - Linux
+
+**Note**: You need the latest Docker version installed.
+
+You might need to pull **onlyoffice/documentserver** image:
+
+```bash
+sudo docker pull onlyoffice/documentserver
+```
+
+### Create develop image
+To create a image with the ability to include external non-minified sdkjs code, use the following command from the dockerfile directory:
+```bash
+sudo docker build -t your_imageName .
+```
+**Note**: The dot at the end is required.
```

Comment:
Not sure that this note is needed, it's just docker syntax, user should execute command as stated in README